content/_help/manage-your-account/overview._en.md

```diff
@@ -2,8 +2,8 @@
 layout: help
 title: Manage your account
 category: manage-your-account
-meta_title: Overview
 permalink: /help/manage-your-account/overview/
+meta_title: Manage your account
```

Comment:
I'm not sure what NetlifyCMS will let you do, but the `meta_title` field only seems to be necessary if the text is different than the page's default title. In this case, we're making it the same.

Aside: If the goal is to get rid of those disparities, maybe we get rid of the `meta_title` functionality altogether? It seems to be used exclusively for making this "Overview" link text different than the page's default title.

Reply:
I agree with you that these landing pages (as this isn't the only page that serves as a "disambiguation" page) are not useful! That said, if we change one, we ought to change them all because it's a standard pattern on the help site. IMO, this is out of scope for these tactical changes, but one I'd welcome brainstorming on in the future. 

Regarding meta_title: I don't really understand Netlify's incorporation of these but all I saw in the meta title was "Overview" so I changed it :X